benchmark/README.md

```diff
@@ -9,7 +9,7 @@ Most of the http benchmarks require [`wrk`][wrk] and [`ab`][ab] (ApacheBench) be
 These may be available through your preferred package manager.
 
 If they are not available:
-- `wrk` may easily built [from source](https://github.com/wg/wrk) via `make`.
+- `wrk` may be easily built [from source](https://github.com/wg/wrk) via `make`.
```

Comment:
Haha, oops. While you are at it, could you change the source link to be `[from source][wrk]`? Thanks!